tools/cuds-prioritized-attribution/composer/dependencies/project_label_credit_data.py

```diff
@@ -0,0 +1,384 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+query = """
```

Comment:
The reason that it's used as a Python file, not SQL, is so that we can inject in variables that are instantiated in the main.py file. Open to other ideas of how to do this, if you have them, but we need to provide the SQL files with values at runtime.

Reply:
look into using jinja templates.